react/features/welcome/components/WelcomePageSideBar.native.js

```diff
@@ -25,11 +26,6 @@ import styles, { SIDEBAR_AVATAR_SIZE } from './styles';
  */
 const PRIVACY_URL = 'https://jitsi.org/meet/privacy';
 
-/**
- * The URL at which the user may send feedback.
- */
-const SEND_FEEDBACK_URL = 'mailto:support@jitsi.org';
```

Comment:
We should still have a button for this outside of the webview, I think. Maybe at the bottom?

Reply:
We agreed to make it part of the faq, I think that's better, especially now that we made the faq url configurable (e.g. for branding)